ts/parts/LogarithmicAxis.ts

```diff
@@ -19,180 +26,267 @@ import H from './Globals.js';
 declare global {
     namespace Highcharts {
         interface Axis {
-            _minorAutoInterval?: (null|number);
-            getLogTickPositions(
-                interval: number,
-                min: number,
-                max: number,
-                minor?: boolean
-            ): Array<number>;
+            /** @deprecated */
             lin2log(num: number): number;
+            /** @deprecated */
```

Comment:
These are marked deprecated, but are used heavily internally?

Reply:
Yes. It is necessary to support the wrapping in v8.